### PR TITLE
Added libfpx recipe

### PR DESCRIPTION
**Pull request by Abogical**
This is to finish the gci task: https://codein.withgoogle.com/dashboard/task-instances/6461249269268480/

This will add the recipe for the library that handles FlashPIX images.

Also corrected the errors pointed out by the last pull request I made.

### CODE REVIEW

File: media-libs/libfpx/libfpx-1.3.1.recipe [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+	make test
+}
+
+REQUIRES_devel="
+	libfpx
+"
+
+PROVIDES_devel="
+	libfpx_devel=$portVersion
+	devel:libfpx=$portVersion
```
Comment (diversys) [inline media-libs/libfpx/libfpx-1.3.1.recipe]: Should be
libfpx$_devel = $portVersion
devel:llibfpx = $portVersion

File: media-libs/libfpx/libfpx-1.3.1.recipe [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+	packageEntries devel $developDir
+}
+
+TEST()
+{
+	make test
+}
+
+REQUIRES_devel="
+	libfpx
```
Comment (diversys) [inline media-libs/libfpx/libfpx-1.3.1.recipe]: libfpx == $portVersion base

**Comment by diversys** [comment]
Do not close this PR when doing fixes, instead use git push force. See https://help.github.com/articles/changing-a-commit-message/#amending-older-or-multiple-commit-messages

File: media-libs/libfpx/libfpx-1.3.1.recipe [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+	
+	packageEntries devel $developDir
+}
+
+TEST()
+{
+	make test
+}
+
+REQUIRES_devel="
```
Comment (humdingerb) [inline media-libs/libfpx/libfpx-1.3.1.recipe]: The _devel blocks come after the regular REQUIRES and PROVIDES.

File: media-libs/libfpx/libfpx-1.3.1.recipe [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+	libfpx == $portVersion base
+"
+
```
Comment (humdingerb) [inline media-libs/libfpx/libfpx-1.3.1.recipe]: No blank line.

File: media-libs/libfpx/libfpx-1.3.1.recipe [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+REQUIRES="haiku"
+
+BUILD_PREREQUIRES="
+	haiku_devel
+	cmd:g++
+	cmd:make
+	cmd:awk
+"
+
```
Comment (humdingerb) [inline media-libs/libfpx/libfpx-1.3.1.recipe]: Just one blank line.

File: media-libs/libfpx/libfpx-1.3.1.recipe [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+SOURCE_URI="http://www.imagemagick.org/download/delegates/libfpx-1.3.1-6.tar.xz"
+CHECKSUM_SHA256="8ea8df5454f1a96c8a49446b7a0d3f7374ce87c501f90e64f4d6ea2d735b0c94"
+SOURCE_DIR="libfpx-1.3.1-6"
+ARCHITECTURES="?x86 ?x86_64 x86_gcc2 ?arm"
+
+PROVIDES="
+	libfpx = $portVersion
+	lib:libfpx = $portVersion
+"
+
```
Comment (humdingerb) [inline media-libs/libfpx/libfpx-1.3.1.recipe]: No blank line.

File: media-libs/libfpx/libfpx-1.3.1.recipe [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+REQUIRES="haiku"
```
Comment (humdingerb) [inline media-libs/libfpx/libfpx-1.3.1.recipe]: Put haiku in a new line

File: media-libs/libfpx/libfpx-1.3.1.recipe [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+
+BUILD_PREREQUIRES="
+	haiku_devel
```
Comment (humdingerb) [inline media-libs/libfpx/libfpx-1.3.1.recipe]: Put haiku_devel into a "BUILD_REQUIRES" block above.

File: media-libs/libfpx/libfpx-1.3.1.recipe [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+	cmd:g++
+	cmd:make
+	cmd:awk
```
Comment (humdingerb) [inline media-libs/libfpx/libfpx-1.3.1.recipe]: Sort alphabetically.

File: media-libs/libfpx/libfpx-1.3.1.recipe [hunk 10]
```diff
@@ -0,0 +1,56 @@
...
+and is based on source code obtained from the Digital Imaging Group Inc. (currently known as International \
+Imaging Industry Association or i3a)."
+HOMEPAGE="http://www.i3a.org"
+COPYRIGHT="1999 Digital Imaging Group, Inc."
+LICENSE="Flashpix"
+REVISION="1"
+SOURCE_URI="http://www.imagemagick.org/download/delegates/libfpx-1.3.1-6.tar.xz"
+CHECKSUM_SHA256="8ea8df5454f1a96c8a49446b7a0d3f7374ce87c501f90e64f4d6ea2d735b0c94"
+SOURCE_DIR="libfpx-1.3.1-6"
+ARCHITECTURES="?x86 ?x86_64 x86_gcc2 ?arm"
```
Comment (humdingerb) [inline media-libs/libfpx/libfpx-1.3.1.recipe]: Blank line before ARCHITECTURES.

File: media-libs/libfpx/libfpx-1.3.1.recipe [hunk 11]
```diff
@@ -0,0 +1,56 @@
+SUMMARY="FlashPIX Open Source Toolkit"
```
Comment (humdingerb) [inline media-libs/libfpx/libfpx-1.3.1.recipe]: "FlashPIX open source toolkit" looks better, also in DESCRIPTION

**Comment by Abogical** [comment]
All comments addressed, it's good to go,

I hope.

**Comment by korli** [comment]
Well the commits were not squashed. Please don't merge such PR next time.

**Pull request merged**